### DISCUSSION TITLE
Define `STATIC_FILES` in `LonaView` like in `Widget`

### DISCUSSION THREAD

**Discussion by maratori** [lona-web-org/lona#95] [Feature Request]
I want to add style sheets to my view. Now I've found following solutions.

1. Replace default styles
```py
app.add_static_file("lona/style.css", path="my-style.css")
```

2. Define fake widget (lona will do it's magic and incorporate all styles from all widgets to HTML)
```py
class Fake(Widget):
    STATIC_FILES = [
        StyleSheet("my-style.css", "my-style.css")
    ]
```

3. Add `link` manually
```py
app.add_static_file("my-style.css", path="my-style.css")

@app.route("/")
class MyView(LonaView):
    def handle_request(self, request):
        return HTML(
            "<link href='/static/my-style.css' rel='stylesheet'>",
        )
```

All solutions above looks hackish.
Let's add `LonaView.STATIC_FILES` like it is in widgets.
So I can just use following code to add style to my view.
```py
@app.route("/")
class MyView(LonaView):
    STATIC_FILES = [
        StyleSheet("my-style.css", "my-style.css")
    ]

    def handle_request(self, request):
        ...
```

**Comment by fscherf** [comment]
@maratori Great idea! I never thought of that. Could you provide a pr with a prototype and update the documentation?

Edit: 2 and 3 are hackish, 1 is ok.

**Reply by maratori** [reply]
Created #134

**Reply by fscherf** [reply]
@maratori great! :)
